.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,16 +5,28 @@
 ## Provide some background on the changes
 > Give details ex. Security patching, content update, more API pods etc
 
-## If you are releasing a new version of notify, what components are you updating
+## If you are releasing a new version of Notify, what components are you updating
 - [ ] API
 - [ ] Admin
 - [ ] Documentation
-- [ ] Document API
-- [ ] Document UI
+- [ ] Document download API
+- [ ] Document download frontend
 
 ## Checklist if releasing new version:
-- [ ] I made sure that both API and Admin changes are present in Notify
-- [ ] I have checked if the docker images I am referencing exist - ex: https://gallery.ecr.aws/v6b8u5o6/notify-admin
+- [ ] I made sure that the changes are as expected in [Notify staging](https://staging.notification.cdssandbox.xyz/)
+- [ ] I have checked if the docker images I am referencing exist
+    - [ ] [api lambda](https://ca-central-1.console.aws.amazon.com/ecr/repositories/private/296255494825/notify/api-lambda?region=ca-central-1) (requires Notification-Production / AdministratorAccess login)
+    - [ ] [api k8s](https://gallery.ecr.aws/v6b8u5o6/notify-api)
+    - [ ] [admin](https://gallery.ecr.aws/v6b8u5o6/notify-admin)
+    - [ ] [documentation](https://gallery.ecr.aws/v6b8u5o6/notify-documentation)
+    - [ ] [document download API](https://gallery.ecr.aws/v6b8u5o6/notify-document-download-api)
+    - [ ] [document download frontend](https://gallery.ecr.aws/v6b8u5o6/notify-document-download-frontend)
```

Comment:
Quite handy, especially after the changes in notification pr-bot: it's a good thing to check once it is merged.